pkg/cmd/server/start/admission.go

```diff
@@ -41,6 +41,7 @@ var (
 
 func init() {
 	defaultOffPlugins.Insert("AlwaysPullImages")
+	defaultOffPlugins.Insert("JenkinsBootstrapper")
```

Comment:
moving @smarterclayton's comment to the right thread:

> I don't consider the admission on off interface acceptable. The enabled flag was specifically put in to give all cluster admins a single, explicable point of control for jenkins auto creation. Admission fiddling is not usable, discoverable, or safe.

so are we going to remove the "disable random admission controllers" feature?  because it's pretty confusing if we're going to have it and then not let people use it.